packages/driver/src/multi-domain/communicator.ts

```diff
@@ -105,7 +105,8 @@ export class PrimaryDomainCommunicator extends EventEmitter {
    * @param {any} data - any meta data to be sent with the event.
    */
   toSpecBridge (event: string, data?: any) {
-    this.crossDomainDriverWindow.postMessage({
+    // If there is crossDomainDriverWindow, there is no need to send the message.
+    this.crossDomainDriverWindow?.postMessage({
```

Comment:
With this change we can call `toSpecBridge` before the spec bridge has been initialized. If it doesn't exist, thats fine (at least in the case where we're telling it the state of 'duringUserTestExecution')